prdoc/pr_4595.prdoc

```diff
@@ -0,0 +1,15 @@
+title: "Remove `elastic-scaling-experimental` feature flag"
+
+doc:
+  - audience: Node Dev
+    description: |
+      The feature was masking the ability of collators to respond with `CollationWithParentHeadData`
+      to validator collation fetch requests. This is no longer needed as validators 
+      have been upgraded to support the new response. `CollationWithParentHeadData` however
```

Comment:
this is not really true for all validators. I would add a section for Node Operator encouraging to upgrade and saying what will happen if they don't

Reply:
done!